src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -953,6 +956,42 @@ private void handleGetGenericSubObj(RoutingContext ctx, String subObj) {
     }
   }
 
+  private void handleGetPieceById(RoutingContext ctx) {
+
+    logger.info("handleGetPiecesById got: " + ctx.request()
+      .path());
+    String pieceId = ctx.request().getParam(ID);
+    JsonObject body;
+    try {
+      if (NON_EXIST_VENDOR_ID.equals(pieceId)) {
```

Comment:
I believe first 2 `if` branches should be removed because they are copied from vendors and looks like are not relevant to pieces...

Reply:
Sure, removed it.